library/src/scala/compiletime/package.scala

```diff
@@ -45,11 +45,13 @@ package object compiletime {
     *  // ("code: scala.Predef.identity("foo")", identity("foo"))
     *  ```
     *
-    * @note only by-name arguments will be displayed as "code".
+    *  The formatting of the code is not stable across version of the compiler.
+    *
+    * @note only `inline` arguments will be displayed as "code".
     *       Other values may display unintutively.
     */
-    transparent inline def code (inline args: Any*): String =
-      ${ dotty.internal.CompileTimeMacros.codeExpr('self, 'args) }
+    transparent inline def code (inline args: Any*): String = ???
```

Comment:
Add a comment pointing to where this is implemented in the compiler

Reply:
I added the comments and also used `error(...)` instead of `???` to have a compile-time failure if the compiler misses it as an intrinsic and just inlines it.